opentelemetry/proto/common/v1/common.proto

```diff
@@ -66,9 +66,20 @@ message KeyValue {
 }
 
 // InstrumentationLibrary is a message representing the instrumentation library information
-// such as the fully qualified name and version. 
+// such as the fully qualified name and version.
+// InstrumentationLibrary is wire-compatible with InstrumentationScope for binary
+// Protobuf format.
+// This message is deprecated and will be removed on June 15, 2022.
 message InstrumentationLibrary {
```

Comment:
FYI: There is a proto deprecated annotation.

Reply:
Add this as first line:

```proto
message InstrumentationLibrary {
  option deprecated = true;

   ...
}
```

Reply:
Added.